src/dev/code_coverage/ingest_coverage/__tests__/enumerate_patterns.test.js

```diff
@@ -21,30 +21,24 @@ describe(`enumeratePatterns`, () => {
       new Map([['x-pack/plugins/screenshotting', ['kibana-screenshotting']]])
     );
 
-    expect(actual).toHaveProperty(
-      '0',
-      expect.arrayContaining([
-        'x-pack/plugins/screenshotting/server/browsers/extract/unzip.ts kibana-screenshotting',
-      ])
+    expect(actual.flat()).toContain(
+      'x-pack/plugins/screenshotting/server/browsers/extract/unzip.ts kibana-screenshotting'
     );
   });
   it(`should resolve src/plugins/charts/common/static/color_maps/color_maps.ts to kibana-app`, () => {
     const actual = enumeratePatterns(REPO_ROOT)(log)(
       new Map([['src/plugins/charts/common/static/color_maps', ['kibana-app']]])
     );
 
-    expect(actual[0][0]).toBe(
+    expect(actual.flat()).toContain(
       'src/plugins/charts/common/static/color_maps/color_maps.ts kibana-app'
     );
   });
   it(`should resolve x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_flyout/translations.ts to kibana-security`, () => {
-    const short = 'x-pack/plugins/security_solution';
+    const short =
+      'x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_flyout';
     const actual = enumeratePatterns(REPO_ROOT)(log)(new Map([[short, ['kibana-security']]]));
 
-    expect(
-      actual[0].includes(
-        `${short}/public/common/components/exceptions/edit_exception_flyout/translations.ts kibana-security`
-      )
-    ).toBe(true);
+    expect(actual.flat()).toContain(`${short}/translations.ts kibana-security`);
```

Comment:
These tests depended on sort order - instead of by index I checked if the item's included.